packages/browser/src/browser/__tests__/inspector.integration.test.ts

```diff
@@ -55,7 +56,51 @@ describe('Inspector', () => {
 
     await deliveryPromise
 
-    expect(enrichedFn).toHaveBeenCalledTimes(1)
+    expect(enrichedFn).toHaveBeenCalledTimes(2)
     expect(deliveredFn).toHaveBeenCalledTimes(1)
   })
+
+  it('emits message failure event', async () => {
```

Comment:
Nitpick: Seems like this is testing a few things, we might want to change the name to "emits expected events" if we aren't breaking it down into subtests

Reply:
Other things are not being tested for, they are there to act as an "experiment control"

Reply:
FYI, other tests in the suite already provide that "control" through their assertions, so my preference is to remove assertions that aren't directly relevant to the test, as they make the test less focused and more brittle. /my 2 cents

Reply:
Ok. Removed